website/content/docs/api-clients/client-agent.mdx

```diff
@@ -172,20 +172,17 @@ By default, it is located in the following directory:
 
 1. Change the configuration settings, and save the file.
 
+   <Note>
+
    You must restart the Client Agent to update some configuration settings.
    However, when you restart the Client Agent, it closes any existing sessions.
    Other configuration settings can be updated by only reloading the configuration file, which does not affect any existing sessions.
 
-1. Either restart the Client Agent or reload the configuration file.
-
-   You can update any configuration value by restarting the Client Agent with the following commands, however it will close any existing sessions:
+   </Note>
 
-   ```shell-session
-   $ sudo launchctl stop com.hashicorp.boundary.boundary-client-agent
-   $ sudo launchctl start com.hashicorp.boundary.boundary-client-agent
-   ```
+1. Either reload the configuration file or restart the Client Agent.
 
-   Alternatively, you can update the following configuration values by reloading the configuration file, which will not disrupt any existing sessions:
+   The following configuration values can be changed by reloading the configuration file, which will not disrupt any existing sessions:
```

Comment:
```suggestion
   You can change the following configuration values by reloading the configuration file, which will not disrupt any existing sessions:
```

Reply:
Suggest we make this active voice ("you can change" vs "can be changed") for clarity about who performs the action.